pkg/topology/kademlia/internal/metrics/metrics.go

```diff
@@ -208,7 +244,7 @@ func (cs *Counters) snapshot(t time.Time) *Snapshot {
 		SessionConnectionDuration:  sessionConnDuration,
 		SessionConnectionDirection: cs.sessionConnDirection,
 		LatencyEWMA:                cs.latencyEWMA,
-		IsReachable:                cs.isReachable,
+		ReachabilityStatus:         cs.ReachabilityStatus,
```

Comment:
nitpick, for the same of uniformity (like below), maybe `Reachability` is enough?